src/app/tests/suites/certification/Test_TC_DRLK_2_6.yaml

```diff
@@ -107,40 +107,40 @@ tests:
           error: INVALID_COMMAND
 
     - label:
-          "Step 5: TH sends Get Holiday Schedule Command to DUT with Invalid
-          HolidayIndex as 15"
+          "Step 5: TH sends GetHolidaySchedule Command to DUT with Invalid
+          HolidayIndex > number_holiday_sch_supported ."
       PICS: DRLK.S.F0b && DRLK.S.C12.Rsp && DRLK.S.C12.Tx
       command: "GetHolidaySchedule"
       arguments:
           values:
               - name: "HolidayIndex"
-                value: 15
+                value: NumberOfHolidaySchedulesSupportedValue + 1
       response:
           values:
               - name: "HolidayIndex"
-                value: 15
+                value: NumberOfHolidaySchedulesSupportedValue + 1
```

Comment:
Don't check the index, it's not defined how it would get returned on error, and it's not required by test plan. Only check status.